tools/flaky-test-reporter/result.go

```diff
@@ -130,7 +130,7 @@ func flakyRateAboveThreshold(rd *RepoData) bool {
 // createArtifactForRepo marshals RepoData into json format and stores it in a json file,
 // under local artifacts directory
 func createArtifactForRepo(rd *RepoData) error {
-	outFilePath := path.Join(prow.GetLocalArtifactsDir(), rd.Config.Repo+".json")
+	outFilePath := path.Join(prow.GetLocalArtifactsDir(), rd.Config.Repo, rd.Config.Name+".json")
```

Comment:
I would prefer this to be a function instead of 2 separate function defining this output directory separately:
prow.GetLocalArtifactsDir(), rd.Config.Repo

Reply:
also, move it to after the err. No use getting this var, if we are goign to return err

Reply:
Do you mean that I should move directory creation from main to this function?